src/style.css

```
@@ -1,3 +1,96 @@
 body {
   margin: 0;
+  font-family: Roboto, 'Times New Roman', Times, serif;
+}
+
+.header {
+  background-color: #fff;
+  text-align: center;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav {
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+  width: 40px;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    height: 60px;
+    width: 1024px;
+  }
+}
+
+@media (min-width: 1025px) and (max-width: 1200px) {
+  .header {
+    height: 60px;
+    width: 1200px;
+  }
+}
+
+.link {
+  height: 60px;
+  width: 100%;
+  margin: auto;
+  display: flex;
+}
+
+.box:not(:first-child) {
+  margin-left: 20px;
+}
+
+.box a {
+  color: #000;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  display: inline-block;
+  text-decoration: none;
+  align-content: center;
+  block-size: 20px;
+  position: relative;
+  padding-top: 23px;
+  padding-bottom: 22px;
+}
+
+.box-1 a {
```

Use only class selector
```suggestion
.nav__link {
```